subprojects/docs/src/docs/userguide/base_plugin.adoc

```diff
@@ -33,7 +33,7 @@ include::{samplesPath}/userguide/basePlugin/build.gradle[tag=apply-base-plugin]
 == Task
 
 `clean` — type: `Delete`::
-Deletes the build directory and everything in it, i.e. the path specified by the link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:getBuildDir()[Project.getBuildDir()] project property.
+Deletes the build directory and everything in it, i.e. the path specified by the link:{groovyDslPath}/org.gradle.api.Project.html#org.gradle.api.Project:buildDir[Project.getBuildDir()] project property.
```

Comment:
If we're explicitly specifying the text for the link now, can we not use property syntax? I.e. `Project.buildDir` or just `buildDir`.